microsoftgraph/microsoft-graph-explorer-v4
# Fix: Response headers copy icon

## Overview
Solves #1389 

### Demo
![image](https://user-images.githubusercontent.com/45680252/152161769-c8f8cf8e-f028-4cb0-b1d4-bdeed1f386b6.png)

![image](https://user-images.githubusercontent.com/45680252/152161831-40cad8f2-3a84-49a2-9ca3-5ece434da2f0.png)

Notice that the icon changes to a checkmark when clicked

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

Run any sample query
Go to Response Headers tab
Click on the copy icon on the far right
Notice the behavior change
